.github/workflows/debian-draft-executable-release.yml

```diff
@@ -28,20 +31,20 @@ jobs:
           TAG_NAME: ${{ steps.get-tag.outputs.TAG_NAME }}
           REPO_NAME: ${{ github.repository }}
           
-  notify-complete-fail:
-    if: ${{ failure() || cancelled() }}
-    needs: [ debian-executable-release ]
-    name: Notify Release Failed
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v2
-      - name: Slack Notification
-        uses: rtCamp/action-slack-notify@v2
-        env:
-          SLACK_WEBHOOK: ${{ secrets.ALERT_SLACK_WEB_HOOK }}
-          SLACK_COLOR: "#ff3333"
-          SLACK_USERNAME: CLI Release Bot
-          SLACK_ICON_EMOJI: ":ship:"
-          SLACK_TITLE: "Twilio Cli"
-          SLACK_MESSAGE: 'Debian Executable Release Failed'
-          MSG_MINIMAL: actions url
\ No newline at end of file
+  # notify-complete-fail:
```

Comment:
Why we commented the notification part? Can we not specify draft release debian executable is failed?

Reply:
Since it's the draft release which would be monitored while running, instead of sending out multiple notifications I preferred sending one after the workflow had finished. Do you think we would need notifications, I can enable them all.

Reply:
If we monitor, removing make sense.